<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Capacity</class>
 <widget class="QWidget" name="Capacity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>容量</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
	font: 16pt &quot;Arial&quot;;
	border-radius:5px
}
QPushButton:hover{
	font: 20pt &quot;Arial&quot;;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item row="7" column="3">
        <widget class="QPushButton" name="btnNum6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>6</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="3">
        <widget class="QGroupBox" name="groupBoxUp">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="displayUp">
              <property name="styleSheet">
               <string notr="true">font-size:40pt ;
text-align:left;
</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBoxUp">
            <item>
             <property name="text">
              <string>毫升</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方厘米</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>升</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方米</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>茶匙（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>汤匙（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>液盎司（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>杯（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>品脱（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>夸脱（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>加仑（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方英寸</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方英尺</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方码</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>茶匙（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>汤匙（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>液盎司（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>品脱（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>夸脱（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>加仑（英制）</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="btnNum5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>5</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="btnNum1">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>1</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btnDel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(246, 246, 246);
}
QPushButton:!hover{
	background-color:rgb(249, 249, 249);
	font: 12pt &quot;Arial&quot;;
}</string>
         </property>
         <property name="text">
          <string>⌫</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="btnNum3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>3</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="3">
        <widget class="QPushButton" name="btnNum9">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>9</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="btnNum0">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="btnNum2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="btnNum8">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>8</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="3">
        <widget class="QPushButton" name="btnPeriod">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>.</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QPushButton" name="btnNum7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>7</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="btnClearAll">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(246, 246, 246);
}
QPushButton:!hover{
	background-color:rgb(249, 249, 249);
	font: 12pt &quot;Arial&quot;;
}</string>
         </property>
         <property name="text">
          <string>CE</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QPushButton" name="btnNum4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:hover{
	background-color: rgb(252, 252, 252);
}
QPushButton:!hover{
	background-color:rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>4</string>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QGroupBox" name="groupBoxDown">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="displayDown">
              <property name="styleSheet">
               <string notr="true">font-size:40pt ;
text-align:left;
</string>
              </property>
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QComboBox" name="comboBoxDown">
            <item>
             <property name="text">
              <string>毫升</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方厘米</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>升</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方米</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>茶匙（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>汤匙（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>液盎司（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>杯（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>品脱（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>夸脱（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>加仑（美制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方英寸</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方英尺</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>立方码</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>茶匙（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>汤匙（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>液盎司（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>品脱（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>夸脱（英制）</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>加仑（英制）</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
